t/19-isotovideo-command-processing.t

```diff
@@ -410,6 +411,16 @@ subtest 'No readable JSON' => sub {
     is($command_handler->loop, 0, 'Loop was stopped');
 };
 
+subtest 'Stopping the command server' => sub {
+    # A fake server and port is enough to exercise the code path
+    $command_handler->cmd_srv_port(1);
+    $command_handler->cmd_srv_process(process(sub { })->start);
```

Comment:
This test might fail if the sub process already existed when `stop_server` is called (and thus `$self->cmd_srv_process->is_running` returns false). Maybe it makes sense to invoke e.g. `sleep infinity` here so the process keeps idling until it is stopped.